### PR TITLE
feat: add cosmos-db env variables for staging asset-services workers

### PR DESCRIPTION
**Pull request by billguo99**
Adding environment variables for asset-services-staging-workers for all organizations for the Wallet Address Cosmos DB.

### CODE REVIEW

File: docker-compose.bhutan.yaml [hunk 1]
```diff
@@ -69,6 +69,9 @@ services:
       TWILIO_AUTH_TOKEN:
       TWILIO_VERIFY_SERVICE_SID:
       TWILIO_MESSAGING_SERVICE_SID:
+      BHUTAN_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING:
+      BHUTAN_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME:
+      BHUTAN_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME:
```
Comment (jdvlio) [inline docker-compose.bhutan.yaml]: Rather use shell interpolation here:
```suggestion
      WALLET_ADDRESS_DB_CONNECTION_STRING: ${BHUTAN_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING}
      WALLET_ADDRESS_DB_DATABASE_NAME: ${BHUTAN_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME}
      WALLET_ADDRESS_DB_COLLECTION_NAME: ${BHUTAN_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME}
```
This way all the containers use the same environment variable but they're value will be derived from a separate entry in the `.env` file.  You generally want your configuration to be as uniform as possible.

**Review by sonasi — APPROVED** [review]

File: docker-compose.palau.yaml [hunk 2]
```diff
@@ -69,6 +69,9 @@ services:
       TWILIO_AUTH_TOKEN:
       TWILIO_VERIFY_SERVICE_SID:
       TWILIO_MESSAGING_SERVICE_SID:
+      PALAU_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING:
+      PALAU_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME:
+      PALAU_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME:
```
Comment (jdvlio) [inline docker-compose.palau.yaml]:
```suggestion
      WALLET_ADDRESS_DB_CONNECTION_STRING: ${PALAU_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING}
      WALLET_ADDRESS_DB_DATABASE_NAME: ${PALAU_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME}
      WALLET_ADDRESS_DB_COLLECTION_NAME: ${PALAU_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME}
```

File: docker-compose.yaml [hunk 3]
```diff
@@ -108,9 +108,9 @@ services:
       TWILIO_AUTH_TOKEN:
       TWILIO_VERIFY_SERVICE_SID:
       TWILIO_MESSAGING_SERVICE_SID:
-      WALLET_ADDRESS_DB_CONNECTION_STRING:
-      WALLET_ADDRESS_DB_DATABASE_NAME:
-      WALLET_ADDRESS_DB_COLLECTION_NAME:
+      NTLS_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING:
+      NTLS_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME:
+      NTLS_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME:
```
Comment (jdvlio) [inline docker-compose.yaml]:
```suggestion
      WALLET_ADDRESS_DB_CONNECTION_STRING: ${NTLS_STAGING_WALLET_ADDRESS_DB_CONNECTION_STRING}
      WALLET_ADDRESS_DB_DATABASE_NAME: ${NTLS_STAGING_WALLET_ADDRESS_DB_DATABASE_NAME}
      WALLET_ADDRESS_DB_COLLECTION_NAME: ${NTLS_STAGING_WALLET_ADDRESS_DB_COLLECTION_NAME}
```

**Review by jdvlio — CHANGES_REQUESTED** [review]

**Review by jdvlio — APPROVED** [review]

**Review by jdvlio — APPROVED** [review]
:+1:

**Pull request merged**